devtools/tx-tests/README.md

```diff
@@ -4,13 +4,17 @@ Create file `devtools/tx-tests/src/config/type_ids.toml`
 ```
 touch devtools/tx-tests/src/config/type_ids.toml
 ```
+You can fill type_ids.toml by running `cargo run -- tx -n test -i`.  
```

Comment:
But in most cases, there is no need to manually fill in the file, as it will be automatically filled in when running cases.

Reply:
And there are also very few instances to run`cargo run -- tx -n test -i`.